pkg/utils/config-manager/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Build Stage
+FROM golang:1.19.0 AS build
```

Comment:
I'd suggerst to use the same golang version as for other images

```suggestion
FROM golang:${GO_VERSION}-buster as build
```

Reply:
done